apps/daffio/src/index.html

```diff
@@ -2,10 +2,11 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>Daff.io</title>
+  <title>Daffodil</title>
```

Comment:
Daffodil - a frontend toolkit for ecommerce.

Reply:
Updated in 3af15b9